integration_tests/integration/confirmAddedToRoll.spec.ts

```diff
@@ -29,17 +29,17 @@ context('Confirm Added To Roll', () => {
     const confirmArrivalPage = ConfirmArrivalPage.goTo(expectedArrival.id)
     confirmArrivalPage.continue().click()
 
-    const imprisonmentStatusPage = ImprisonmentStatusPage.goTo(expectedArrival.id)
+    const imprisonmentStatusPage = SexPage.goToWithRedirect(expectedArrival.id)
```

Comment:
this test wasn't correct previously I don't think - don't need to goTo after clicking a link - just need to verify you've landed on the correct page. 

`const imprisonmentStatusPage = Page.verifyOnPage(ImprisonmentStatusPage)`